src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -47,19 +47,19 @@ enum AddressSuggestionRadioButton {
 }
 
 const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
-  verifiedAddressResult,
+  verifyAddress,
   onChosenAddress,
   onClose,
 }) => {
   const [modalType, setModalType] = useState<ModalType | null>(null)
   const [addressOptions, setAddressOptions] = useState<AddressOption[]>([])
-  const [selectedAddressKey, setSelectedAddressKey2] = useState<string | null>(
+  const [selectedAddressKey, setSelectedAddressKey] = useState<string | null>(
     null
   )
 
-  const setSelectedAddressKey = (key: string) => {
-    setSelectedAddressKey2(key)
-  }
+  const emptyDiv: JSX.Element = (
+    <div data-testid="emptyAddressVerification"></div>
```

Comment:
This allows our tests to await the initial render when the `useEffect` hooks are called, which in turn allows them to make assertions about changes made by those hooks.